packages/web3auth/src/index.ts

```diff
@@ -49,13 +51,30 @@ function web3auth(options: Web3AuthModuleOptions): WalletInit {
         }
       })
 
-      const web3authOptions = {
+      const web3authCoreOptions = {
         ...options,
         ...getChainConfig(currentChain)
       }
 
-      let web3auth = new Web3Auth(web3authOptions)
-
+      let web3auth = new Web3Auth(web3authCoreOptions)
+
+      // There is no exposed z-index setter so we must set through modal events
```

Comment:
geeze that is unforntunate but I commend you on this hackery 🫡 haha

Reply:
Right?! Really surprising they dont have any config for the z-index of the modal. I created a feature request with their team to get this added - https://github.com/Web3Auth/web3auth-web/issues/1018